httpbp/prometheus.go

```diff
@@ -16,6 +16,22 @@ const (
 	endpointLabel   = "http_endpoint"
 )
 
+var ResponseSizeBuckets = []float64{
```

Comment:
I'm not sure we want to export this and make this part of our API, better unexport it (use lower case `responseSizeBuckets`) instead.

also this is used in both request and response, so a better name would be `payloadSizeBuckets`.